versionsupport_v1_21_r1/src/main/java/com/tomkeuper/bedwars/support/version/v1_21_R1/v1_21_R1.java

```diff
@@ -685,9 +690,9 @@ public String getMainLevel() {
     @Override
     public Fireball setFireballDirection(Fireball fireball, Vector vector) {
         EntityFireball fb = ((CraftFireball) fireball).getHandle();
-        fb.d = vector.getX() * 0.1D;
-        fb.e = vector.getY() * 0.1D;
-        fb.f = vector.getZ() * 0.1D;
+        // fb.d = vector.getX() * 0.1D;
```

Comment:
Further testing is needed here

Reply:
https://mappings.dev/1.21/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.html